storage-proofs/src/stacked/proof.rs

```diff
@@ -454,7 +477,7 @@ impl<'a, H: 'static + Hasher, G: 'static + Hasher> StackedDrg<'a, H, G> {
                 trace!("building merkle tree for the original data");
                 data.ensure_data()?;
                 measure_op(CommD, || {
-                    Self::build_tree::<G>(data.as_ref(), tree_d_config.clone())
+                    Self::build_binary_tree::<G>(data.as_ref(), tree_d_config.clone())
```

Comment:
I see that it's not here, just want to be sure it's intentional.

Reply:
yes it is for now

Reply:
Ok, thanks